sig-node/README.md

```diff
@@ -75,7 +75,7 @@ The following [subprojects][subproject-definition] are owned by sig-node:
 - **Owners:**
   - [kubernetes/cri-api](https://github.com/kubernetes/cri-api/blob/master/OWNERS)
   - [kubernetes/kubernetes/staging/src/k8s.io/cri-api](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cri-api/OWNERS)
-  - [kubernetes/kubernetes/staging/src/k8s.io/cri-api/pkg](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cri-api/pkg/OWNERS)
+  - [kubernetes/kubernetes/staging/src/k8s.io/cri-api/pkg](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cri-api/pkg)
```

Comment:
can you please find the proper OWNERS link instead?

Reply:
@vaibhav2107 staging/src/k8s.io/cri-api/pkg/OWNERS file doesn't exist, so I'd suggest to remove that link, but not from README.md

Here is what failed CI test suggests:
```
...
Do not manually edit sig-list.md or README.md files inside the sig folders.
Instead make your changes to sigs.yaml, then run `make`, and then
commit your changes to sigs.yaml and any generated docs.
...
```

Please, follow the suggestion or close the PR if you're not willing to continue with it, thanks!

Reply:
@bart0sh I have removed the link by the process you suggested. Please have a look

Reply:
@vaibhav2107 the check is still failing with the same error:
```
====================================================================
1 mismatches were detected.
Do not manually edit sig-list.md or README.md files inside the sig folders.
Instead make your changes to sigs.yaml, then run `make`, and then
commit your changes to sigs.yaml and any generated docs.
====================================================================
[0;[31](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/community/7755/pull-community-verify/1790255812711550976#1:build-log.txt%3A31)mFAILED   verify-generated-docs.sh	20s
Verifying verify-spelling.sh
SUCCESS  verify-spelling.sh	4s
Verifying verify-steering-election.sh
SUCCESS  verify-steering-election.sh	0s
========================
[0;31mFAILED TESTS
========================
[0;31mhack/../hack/verify-generated-docs.sh
make: *** [Makefile:17: verify] Error 1
```